meta-resin-common/recipes-core/images/resin-image.bb

```diff
@@ -48,6 +48,10 @@ generate_hostos_version () {
     echo "${HOSTOS_VERSION}" > ${DEPLOY_DIR_IMAGE}/VERSION_HOSTOS
 }
 
+read_only_rootfs_hook_append () {
+    sed -i -e "s:^DROPBEAR_RSAKEY_DIR=.*$:DROPBEAR_RSAKEY_DIR=/etc/dropbear:" ${IMAGE_ROOTFS}/etc/default/dropbear
```

Comment:
This has to be here, because the read_only_rootfs_hoos will come after any tasks in the dropbear package, thus overwriting anything we have there.